components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java

```diff
@@ -3852,6 +3860,17 @@ public String getMediationNameFromConfig(String config) {
         return null;
     }
 
+    /**
+     * Check the existence of the mediation policy
```

Comment:
Needs empty line after the method description

Reply:
Fixed with f81bbb7